solax/__init__.py

```diff
@@ -1,6 +1,5 @@
 """Support for Solax inverter via local API."""
 import asyncio
-
```

Comment:
From an initial glance, I can see a few things we will need to change:
- There are a lot of whitespace and non-functional changes. I am referring to changes where you have reordered imports or removed whitespace between imports. These pollute the diff an will have to be removed
- I notice that you added a few logger statements. As this is a library, we should not be logging anything and leave the decision on how to bubble-up errors and information to the program calling the library, so you will have to remove the loggers too.